### ISSUE TITLE
loss does not decrease significantly

### ISSUE THREAD

**Issue by cuiyun-fang** [Nik-V9/AirObject#2]
when running the script train/train_airobj.py, I noticed that the loss did not decrease significantly, which is unexpected. The loss is like the following:
sum_iter = 50, loss = 0.7772274613380432, ploss = 0.010768788866698742, nloss=0.7664586901664734
sum_iter = 100, loss = 0.4678459167480469, ploss = 0.20307928323745728, nloss=0.2647666335105896
sum_iter = 150, loss = 0.6148198843002319, ploss = 0.4320690333843231, nloss=0.18275086581707
sum_iter = 200, loss = 0.3721957206726074, ploss = 0.23799094557762146, nloss=0.13420478999614716
sum_iter = 250, loss = 0.5819605588912964, ploss = 0.5520272850990295, nloss=0.029933243989944458
……
it remains within the range of 0.x and does not show a clear overall downward trend.
Is it normal? Or did I miss some details and make mistakes?
